Source/Core/VideoCommon/PixelShaderGen.h

```diff
@@ -43,7 +43,8 @@ struct pixel_shader_uid_data
   u32 numColorChans : 2;
   u32 rgba6_format : 1;
   u32 dither : 1;
-  u32 pad : 16;
+  u32 uint_output : 1;
```

Comment:
I'm not sure an extra field in the UID is the best way to go about this. The bit is not needed for GL and Vulkan, and just creates duplicate shaders there. We also are going to need a solution for the whole framebuffer fetch vs dual src blend issue too, so whatever we decide there can potentially be used for this as well.